src/libprojectM/Renderer/hlslparser/src/Engine.cpp

```diff
@@ -59,8 +64,31 @@ bool String_EqualNoCase(const char * a, const char * b) {
 #endif
 }
 
+static inline double iss_strtod(const char * in, char ** end) {
```

Comment:
Some third party libraries may already do this:

folly:::Conv https://github.com/facebook/folly/blob/master/folly/docs/Conv.md
boost::lexical_cast https://www.boost.org/doc/libs/1_42_0/libs/conversion/lexical_cast.htm

Reply:
If I understand correctly, using either of those would bring in another dependency, which isn't really a good idea for only this specific thing. I'm also not sure if those libraries support getting the end of the float in the string, which is needed for the parser.
I may be wrong though as I am not familiar with either of those, and I only had a quick glance at the links.

Additionally, as mentioned, the rest of the code does the same thing, which is why I think it's the best course of action.
https://github.com/projectM-visualizer/projectm/blob/fd1d235a059714a3cdf1504d7ffe4ef4c286fe02/src/libprojectM/MilkdropPresetFactory/Parser.cpp#L1285-L1300